tests/xharness/Jenkins/TestSelector.cs

```diff
@@ -82,6 +82,37 @@ class TestSelector {
 			"msbuild",
 			"tests/dotnet",
 		};
+		
+			// disabled by default
+		static readonly Dictionary<string, TestSelection> disabledByDefault = new Dictionary<string, TestSelection> {
+			["mtouch"] =  TestSelection.Mtouch,
+			["mmp"] = TestSelection.Mmp,
+			["bcl"] = TestSelection.Bcl,
+			["bcl-xunit"] = TestSelection.BclXUnit,
+			["bcl-nunit"] = TestSelection.BclNUnit,
+			["mscorlib"] = TestSelection.Mscorlib,
+			["btouch"] = TestSelection.Btouch,
+			["mac-binding-project"] = TestSelection.MacBindingProject,
+			["ios-extensions"] = TestSelection.iOSExtensions,
+			["device"] = TestSelection.Device,
+			["xtro"] = TestSelection.Xtro,
+			["cecil"] = TestSelection.Cecil,
+			["old-simulator"] = TestSelection.OldiOSSimulator,
+			["all"] = TestSelection.All,
+		};
+		
+		static readonly Dictionary<string, TestSelection> enabledByDefault = new Dictionary<string, TestSelection> {
```

Comment:
You can use `DefaultTestSelection` to compute which tests are enabled by default and which aren't, which allows you to remove code duplication (you're specifying which tests are the default ones in multiple places).